e2e/attendance-list.spec.ts

```
@@ -0,0 +1,124 @@
+    
+    if (hasSearch) {
+      await searchInput.fill('doador teste');
+      await expect(searchInput).toHaveValue('doador teste');
+    }
+  });
+
+  test('deve exibir status de chamada do participante', async ({ page }) => {
+    await page.goto('/event/evento-teste/backoffice');
+    
+    // Verifica badges de status
+    const statusBadge = page.locator('.el-tag, .status-badge, .badge').first();
```

ESLint

[error] Replace `'.el-tag,·.status-badge,·.badge'` with `".el-tag,·.status-badge,·.badge"`

(prettier/prettier)